uint8   id
uint8[] predecessor_id
uint8[] context_id
uint8   goal
bool    remove_after_success

string reference_frame

geometry_msgs/Quaternion quaternion
float64[] sigma_dtheta

float64[] Phi_q
float64[] Phi_dq
float64[] Phi_ddq

float64[] mu_w
float64[] mu_s
float64[] sigma_w
float64[] sigma_ws
float64[] sigma_s
